tools/run_tests/run_interop_tests.py

```diff
@@ -220,17 +220,20 @@ def __init__(self):
 
   def cloud_to_prod_args(self):
     return (self.client_cmdline_base + _CLOUD_TO_PROD_BASE_ARGS +
-            ['--use_tls'])
+            ['--use_tls=true'])
 
   def cloud_to_cloud_args(self):
     return (self.client_cmdline_base + _CLOUD_TO_CLOUD_BASE_ARGS +
-            ['--use_tls', '--use_test_ca'])
+            ['--use_tls=true', '--use_test_ca=true'])
 
   def cloud_to_prod_env(self):
     return _SSL_CERT_ENV
 
   def global_env(self):
-    return {}
+    # need to manually copy to each jenkins machine if we run into github
```

Comment:
Currently, global_env (as well as all other *_env  functions) are used to set the environment variable setting to actually run the interop client and interop server (see python), not for running the build_interop_image.sh script. 

So what you are doing here make usage of global_env pretty inconsistent  and I would be against it.